FlintCoreTests/ActionDispatchTests.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ActionDispatchTests.swift
+//  FlintCore
+//
+//  Created by Marc Palmer on 24/08/2018.
+//  Copyright © 2018 Montana Floss Co. Ltd. All rights reserved.
+//
+
+import XCTest
+@testable import FlintCore
+
+class ActionDispatchTests: XCTestCase {
+
+    override func setUp() {
+    }
+
+    override func tearDown() {
+    }
```

Comment:
You should delete these or call `super`. My preference is delete.

Reply:
Yeah.